versioned_docs/version-2021fa/assignment4.md

```diff
@@ -28,6 +28,12 @@ Create a component, Playlist, that contains all of the songs you've added. This
 - Have two input fields, one for title and one for artist.
 - Have a button which allows you to submit the new song and adds to the list.
 
+Some things to remember about your implementation or **you will lose points!**:
+
+- Create a type for your songs! Do not leave it as Object type.
```

Comment:
you might mean `any` instead of `Object`?

Reply:
TS-foo weak